src/dataaccess/GistAdjuvantIhanos.json

```diff
@@ -965,6 +972,13 @@
                     }
                 }
             }
+        },
+        "SourceClinicalNote": {
+            "_ShrId": "788dcbc3-ed18-470c-89ef-35ff91854c7f",
+            "_EntryId": "5110",
```

Comment:
Note 5110 is to order a CT-guided biopsy but this procedure requested is for the endoscopic biopsy. Also, I think we have some confusion caused by a misuse of mCODE entries a little.

Ihanos had 2 biopsies: CT-guided scheduled to be performed on 3 August 2016 and endoscopic scheduled to be performed on 18 July 2016. The notes where those are ordered (in PLAN section), should have the same date as the creation date of the procedure requested. We should have notes corresponding to the day they were performed as well that will be source for procedure performed but we have a separate JIRA to address that I believe.